examples/stats/exporter/stackdriver.py

```diff
@@ -12,51 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
+import os
 import random
+import time
 from opencensus.stats import aggregation as aggregation_module
-from opencensus.stats.exporters import stackdriver_exporter as stackdriver
 from opencensus.stats import measure as measure_module
 from opencensus.stats import stats as stats_module
 from opencensus.stats import view as view_module
-from opencensus.tags import tag_key as tag_key_module
+from opencensus.stats.exporters import stackdriver_exporter as stackdriver_stats
 from opencensus.tags import tag_map as tag_map_module
-from opencensus.tags import tag_value as tag_value_module
```

Comment:
I think it is quite important to ensure that the exporter works with tags as the previous example had. Why this removal?

Reply:
This effort is to make the [basic stats example](https://github.com/census-instrumentation/opencensus-java/blob/master/examples/src/main/java/io/opencensus/examples/stats/StackdriverQuickstart.java) consistent everywhere, but I am happy to rollback to previous version.